javascript_nodejs/50.contoso-cafe-bot/bot.js

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { ActivityTypes, CardFactory } = require('botbuilder');
+const { DialogTurnStatus, DialogSet } = require('botbuilder-dialogs');
+const { LuisRecognizer } = require('botbuilder-ai');
+
+const { onTurnProperty } = require('./dialogs/shared/stateProperties');
+const MainDialog = require('./dialogs/mainDialog');
+const welcomeCard = require('./dialogs/welcome');
```

Comment:
>welcomeCard [](start = 6, length = 11)

Should be CamelCase

Reply:
Oh I see... its a variable. If you look at the file this is being exported from its WelcomeCard there.

---
In reply to: [217213572](https://github.com/Microsoft/BotBuilder-Samples/pull/364#discussion_r217213572) [](ancestors = 217213572)